### ISSUE TITLE
Renaming group

### ISSUE THREAD

**Issue by mfolnovic** [mayope/keycloakmigration#80]
We have a need for renaming groups in Keycloak. This can be done through UI.

From what I can see, this works through same APi as `updateGroup`, where the only change would be the `name`.

This is suggested API (I'll open PR soon):
```
id: rename-group
author: mfolnovic
changes: 
  - renameGroup:
      realm: testRealm
      name: GROUP_A
      newName: NEW_GROUP_A
```

I'm also open to reusing `updateGroup` action:

```
id: rename-group
author: mfolnovic
changes: 
  - updateGroup:
      realm: testRealm
      name: GROUP_A
      newName: NEW_GROUP_A
```

I'm also open to any other suggestion, such as better property name than `newName`. :slightly_smiling_face:

**Comment by klg71** [comment]
Hey @mfolnovic  thanks for contributing to this project! :) I will have a look at your pull request.

**Comment by klg71** [comment]
Hey @mfolnovic. I released your change in version 0.2.65. Could you test it?
